xml/warewulf.xml

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<sect1 xml:id="sec-warewulf-deploy-nodes" xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Deploying compute nodes with &warewulf;</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+<remark>Control node vs control server vs controller host vs warewulf server??
+ I've seen all these terms used... which one will we use?</remark>
+
+ <para>
+  &warewulf; is a node deployment system for &hpc; clusters. The compute nodes are booted
+  over the network using PXE. &warewulf; configures DHCP and TFTP on the &warewulf; server
+  (or control node), and configures the compute nodes using <emphasis>node profiles</emphasis>
+  and <emphasis>&warewulf; overlays</emphasis>.
+ </para>
+ <para>
+  <emphasis>Node profiles</emphasis> group shared node configurations together, such as
+  the container or kernel, overlays, and IPMI settings. New nodes automatically use the
+  <literal>default</literal> node profile. You can also create additional node profiles,
+  for example, if two groups of nodes require different containers.
```

Comment:
containers in the warewulf sense is a node image, but this term was not introduced here.

Reply:
Oh, hmm, thanks for letting me know. But if we're importing it from docker, isn't it also a container in the regular sense? I'm not sure if we really need to explain it.

Reply:
The transport format is an OCI container, but in opposite to a traditional container, the warwulf container transports a kernel and is so bootable.

Reply:
Ah ok, thank you :+1: